### PR TITLE
docs(tabs): control

### PR DESCRIPTION
**Pull request by CristianSpatari**

### CODE REVIEW

**Review by victor-pg — CHANGES_REQUESTED** [review]
1. Your control can't handle `className` and `icon`
2. Remove `states` and `display` parameters
---
image

File: docusaurus/docs-internals/editor-controls/group-controls/tabs.md [hunk 1]
```diff
@@ -407,18 +437,143 @@ If you would like to `align` the label inside the `tabs` on the x-axis, you coul
...
+  className?: string;
+  title?: string;
+  label?: string;
+  position?: number;
+  options?: ControlItem[];
+}
+
+interface Props {
+  tabs: TabItem[];
+}
```
Comment (victor-pg) [inline docusaurus/docs-internals/editor-controls/group-controls/tabs.md:522]: you have only 1 props: `tabsColor`

File: docusaurus/docs-internals/editor-controls/group-controls/tabs.md [hunk 2]
```diff
@@ -407,18 +437,143 @@ If you would like to `align` the label inside the `tabs` on the x-axis, you coul
...
+      }
+    }
+    
+    return {
+      ".brz-container": {
+        display: "none"
+      }
+    }
+  }
+}
```
Comment (victor-pg) [inline docusaurus/docs-internals/editor-controls/group-controls/tabs.md:489]:
```ts
{
  id: "actionTabs",
  type: "tabs",
  style: ({ value }) => {

    if (value && value.value === "tabIcon") {
      return {
        ".brz-container.brz-tab-color": {
          display: "none"
        }
      }
    }

    return {
      ".brz-container.brz-tab": {
        display: "block"
      }
    }
  }
}
```

create similar example for `style` from parameters table

File: docusaurus/docs-internals/editor-controls/group-controls/tabs.md [hunk 3]
```diff
@@ -407,18 +437,143 @@ If you would like to `align` the label inside the `tabs` on the x-axis, you coul
...
+interface Props {
+  tabs: TabItem[];
+}
+
+const Container = (props: Props): JSX.Element => {
+  const { value } = props;
+
+  return (
+    <div className="brz-container" >
+      <a href =${value} rel="blank|self"/>
```
Comment (victor-pg) [inline docusaurus/docs-internals/editor-controls/group-controls/tabs.md:529]: invalid html

File: docusaurus/docs-internals/editor-controls/group-controls/tabs.md [hunk 4]
```diff
@@ -407,18 +437,143 @@ If you would like to `align` the label inside the `tabs` on the x-axis, you coul
...
+    </div>
```
Comment (victor-pg) [inline docusaurus/docs-internals/editor-controls/group-controls/tabs.md:530]: write an example where you render something when some tab is active:
```ts
<div>
{ asd === "tabColor" && <ASdCOmponent/> }
</div>
```

**Review by victor-pg — CHANGES_REQUESTED** [review]

File: docusaurus/docs-internals/editor-controls/group-controls/tabs.md [hunk 5]
```diff
@@ -15,23 +15,24 @@ Example of the `tabs` with position `left`:
...
...                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
```
Comment (victor-pg) [inline docusaurus/docs-internals/editor-controls/group-controls/tabs.md:32]: `tabs` doesn't have `selector` properties, so remove this and don't forget to remove also examples with `selector`

File: docusaurus/docs-internals/editor-controls/group-controls/tabs.md [hunk 6]
```diff
@@ -15,23 +15,24 @@ Example of the `tabs` with position `left`:
...
```
Comment (victor-pg) [inline docusaurus/docs-internals/editor-controls/group-controls/tabs.md:33]: "external" -> "tabExternal"

File: docusaurus/docs-internals/editor-controls/group-controls/tabs.md [hunk 7]
```diff
@@ -407,18 +437,126 @@ If you would like to `align` the label inside the `tabs` on the x-axis, you coul
...
+
+In the example below, a custom `Container` component for the Brizy editor is defined, allowing customization through the injection of custom HTML. To receive the return value, ensure that `config.saveTab` is set to `true`.
+
+```tsx
+import { Brizy } from "@brizy/core";
+import React, { JSX } from "react";
+
+
+interface Props {
+  value: string;
```
Comment (victor-pg) [inline docusaurus/docs-internals/editor-controls/group-controls/tabs.md:504]: invalid key, no options will return the `value`

File: docusaurus/docs-internals/editor-controls/group-controls/tabs.md [hunk 8]
```diff
@@ -407,18 +437,126 @@ If you would like to `align` the label inside the `tabs` on the x-axis, you coul
...
+}
+
+const Container = (props: Props): JSX.Element => {
+  const { value } = props;
+
+  return (
+    <div className="brz-container" >
+      { value === "tabsColor" && <Button/> }
```
Comment (victor-pg) [inline docusaurus/docs-internals/editor-controls/group-controls/tabs.md:512]: prettify

**Review by victor-pg — CHANGES_REQUESTED** [review]
1. image
---
2. image

**Comment by CristianSpatari** [comment]
> 1. image
> 
> 2. image

1. Referitor la ";" chat GPT parca spune ca e ok si nu prea are ceva obiectii.

**Review by victor-pg — CHANGES_REQUESTED** [review]
1. image
---
2. image
---
3.

**Pull request merged**